tutorials/peft-curation-with-sdg/config/sem_dedup_config.yaml

```diff
@@ -10,6 +10,7 @@ embeddings_save_loc: "embeddings"
 embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
 embedding_batch_size: 128
 embedding_max_mem_gb: 20
+input_file_type: "jsonl"
```

Comment:
Same here, lets not add config (and actively remove config) which we have support for here. 

We also probably remove `embedding_max_mem_gb `